ui/src/app/views/notebook-redirect/component.ts

```diff
@@ -130,7 +130,10 @@ export class NotebookRedirectComponent implements OnInit, OnDestroy {
     this.jupyterLabMode = jupyterLabMode === 'true';
     this.setNotebookNames();
 
-    this.loadingSub = this.clusterService.listClusters()
+    this.loadingSub = userProfileStore.asObservable()
```

Comment:
Need to Observe here because `freeTierBillingProjectName` is not yet populated on initial load